eng/scripts/Language-Settings.ps1

```diff
@@ -0,0 +1,93 @@
+$Language = "rust"
+$LanguageDisplayName = "Rust"
+$PackageRepository = "crates.io"
+$packagePattern = "Cargo.toml"
+$MetadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/main/_data/releases/latest/rust-packages.csv"
+$GithubUri = "https://github.com/Azure/azure-sdk-for-rust"
+$PackageRepositoryUri = "https://crates.io/crates"
+
+function Get-AllPackageInfoFromRepo ([string] $ServiceDirectory) {
+    $allPackageProps = @()
+    Push-Location $RepoRoot
+    try {
```

Comment:
Majority of our PS scripts are 2 space indentions, I'd suggest that here as well.

Reply:
I've updated .editorconfig to use 2 spaces for powershell